recipes/uwsgi/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: uwsgi
```

Comment:
Wonder if we should call it `uWSGI` as that's the name (letter casing) they use.